jack_mixer.c

```diff
@@ -676,15 +697,24 @@ channel_abspeak_read(
   }
   else
   {
-    return value_to_db(channel_ptr->abspeak);
+    if (mode == Post_Fader) {
+      return value_to_db(channel_ptr->abspeak);
+    } else if (mode == Pre_Fader) {
+      return value_to_db(channel_ptr->abspeak_pre_fader);
+    }
```

Comment:
I think a ternary expression would be more succinct here:

```c
return value_to_db(mode == PreFader ? channel_ptr->abspeak_prefader : channel_ptr->abspeak);
```

It would also prevent compiler warning because possibly reaching the end of the function without a return:

```
jack_mixer.c: In function 'channel_abspeak_read':
jack_mixer.c:706:1: warning: control reaches end of non-void function [-Wreturn-type]
  706 | }
      | ^
```